application/app/src/test/java/uk/nhs/digital/uec/api/controller/FuzzySearchControllerTest.java

```diff
@@ -131,15 +132,16 @@ public void getServicesByFuzzySearchTestValidationError() throws ValidationExcep
             NAME_PRIORITY,
             ADDRESS_PRIORITY,
             POSTCODE_PRIORITY,
-            PUBLIC_NAME_PRIORITY);
+            PUBLIC_NAME_PRIORITY,
+            null);
```

Comment:
here

Reply:
These changes were already merged, this branch was rebased from master